xbmc/addons/kodi-dev-kit/include/kodi/c-api/addon-instance/pvr/pvr_epg.h

```diff
@@ -559,6 +559,13 @@ extern "C"
 
     /// @brief __0001 0000__ : This EPG entry will be flagged as live.
     EPG_TAG_FLAG_IS_LIVE = (1 << 4),
+
+    /// @brief __0010 0000__ : This flag is reserved for PVR addons internal usage.
+    EPG_TAG_FLAG_ADDON_FLAG_1 = (1 << 5),
+
+    /// @brief __0100 0000__ : This flag is reserved for PVR addons internal usage.
+    EPG_TAG_FLAG_ADDON_FLAG_2 = (1 << 6),
```

Comment:
Maybe it could make sense to add them on end of a bit number:
```cpp
    /// @brief __0001 0000 0000 0000 0000 0000 0000 0000__ : This flag is reserved for PVR addons internal usage.
    EPG_TAG_FLAG_ADDON_FLAG_1 = (1 << 28),

    /// @brief __0010 0000 0000 0000 0000 0000 0000 0000__ : This flag is reserved for PVR addons internal usage.
    EPG_TAG_FLAG_ADDON_FLAG_2 = (1 << 29),

    /// @brief __0100 0000 0000 0000 0000 0000 0000 0000__ : This flag is reserved for PVR addons internal usage.
    EPG_TAG_FLAG_ADDON_FLAG_3 = (1 << 30),

    /// @brief __1000 0000 0000 0000 0000 0000 0000 0000__ : This flag is reserved for PVR addons internal usage.
    EPG_TAG_FLAG_ADDON_FLAG_4 = (1 << 31),
```
So other changes does not mix the flow.

@ksooo what do you thing about to have on end, I'm not 100% sure, was only come in my mind.

Reply:
Thanks for your feedback! I agree, using the highest bits makes more sense to keep space for specific purpose usage.